Jenkinsfile

```diff
@@ -177,6 +215,38 @@ stage('Build without sync') {
         stash includes: 'wrappers/build/**/*.dll', name: 'uwp-wrappers-nosync'
       }
     },
+    'macOS': {
+      nodeWithCleanup('osx') {
+        getArchive()
+
+        dir('wrappers') {
```

Comment:
It bugs me a little that everywhere else we unstash core or core & sync and here we don't 🤔